### ISSUE TITLE
Usersnap Feedback - Is it possible to make the narrative editor re-sizeable? The old .net one [...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#762]
**Sender**: scottowenfletcher@gmail.com
**Comment**: Is it possible to make the narrative editor re-sizeable? The old .net one allowed you to click and drag the bottom right-hand corner to make the box bigger. I'm finding it difficult to edit long entries in such a small space
[Open #461 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-201-a37e4cf6/461)

<a href='https://usersnappublic.s3.amazonaws.com/2019-06-27/22-54/79b2d969-1f9f-4e6c-b503-f016cd7493b5.png'>![Usersnap Feedback - Is it possible to make the narrative editor re-sizeable? The old .net one (...)](https://usersnappublic.s3.amazonaws.com/2019-06-27/22-54/a093ae4d-c910-4ac1-bd9b-69623da77185.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-06-27/22-54/79b2d969-1f9f-4e6c-b503-f016cd7493b5.png'>Download original image</a>

**Browser**: Chrome 75 (Windows 10)
**Referer**: [https://participedia.net/case/5861/edit?full=1](https://participedia.net/case/5861/edit?full=1)
**Screen size**: 1280 x 720 **Browser size**: 1920 x 867

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by jesicarson** [comment]
@ascott I agree with this suggestion, and if it can be squeezed in soon with any other edits to the form, that would be great. Hopefully its an easy update?

**Comment by ascott** [comment]
@jesicarson @scottofletcher  this editor doesn't have an option to resize the editor window, so i would need to add that custom functionality. do we want to support resizing vertically and horizontally? 

a couple other possible solutions: 
- add a toggle/button to automatically resize the editor to the height of the browser window
- add a toggle/button to automatically resize the editor to take over the whole browser window for editing, and a toggle to close/go back to the normal editing mode, sort of like fullscreen photo slideshow

**Comment by jesicarson** [comment]
Horizontally would be the most important. Toggle to height of browser window makes sense if that's simple enough?

**Comment by scottofletcher** [comment]
I need to be able to move between the narrative editor and the other data fields quickly though. for example, in filling out the field 'types of interaction', I need to quickly scroll up to the narrative, see how people participated, scroll back to the field and fill it out. if I could just have that little drag corner and resize it to whatever height/width I want, that would be optimal. if not, then horizontally would be be the next best option. if we had the toggle to height of browser window, what would happen to it if I scrolled up or down? would it remain the same height? height isn't optimal b/c it puts the editing tools slightly out of reach.

**Comment by scottofletcher** [comment]
@ascott: I just realised that the text editor in GitHub (the one I'm writing this in right now) is a perfect match for what I need! it allows vertical adjustment, but if we could have ours w/vertical and horizontal that would be fabulous
